### PR TITLE
docs: added sidebar item

### PR DESCRIPTION
**Pull request by vinckr**
Added Kratos Video Guides Transcripts

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/sidebar.js [hunk 1]
```diff
@@ -73,7 +73,8 @@ module.exports = {
     "guides/login-session",
     "guides/secret-key-rotation",
     "guides/high-availability-ha",
-    "guides/docker"
+    "guides/docker",
+    "guides/kratos-video-tutorials"
```
Comment (aeneasr) [inline docs/sidebar.js:77]: Wouldn't this make more sense in the "Further Reading" section?

Comment (vinckr) [reply docs/sidebar.js:77]: It is not very essential true, I just thought put it there because it directly related to the guides above.

Comment (aeneasr) [reply docs/sidebar.js:77]: Yeah ok makes sense!

Comment (aeneasr) [reply docs/sidebar.js:77]: As we discussed this changed to `.json` format - could you update the PR or make a new one?

**Pull request closed without merging**